ts/session/apis/open_group_api/opengroupV2/OpenGroupServerPoller.ts

```diff
@@ -244,12 +246,14 @@ export class OpenGroupServerPoller {
         if (roomHasBlindEnabled(rooms[0])) {
           const maxInboxId = Math.max(...rooms.map(r => r.lastInboxIdFetched || 0));
 
-          // This only works for servers with blinding capabilities
-          // adding inbox subrequest info
-          subrequestOptions.push({
-            type: 'inbox',
-            inboxSince: { id: isNumber(maxInboxId) && maxInboxId > 0 ? maxInboxId : undefined },
-          });
+          if (Storage.get(SettingsKey.hasBlindedMsgRequestsSetting)) {
```

Comment:
Sorry I have forgotten but does this mean "When user wants to receive blinded message requests" then poll? Or is it the opposite?

Reply:
I can close this based on your comment below